include/pisa/query/algorithm/block_max_wand_query.hpp

```diff
@@ -134,26 +134,20 @@ struct block_max_wand_query {
                     }
                 }
 
-                uint64_t next_jump = max_docid;
-
-                if (pivot + 1 < ordered_cursors.size()) {
```

Comment:
Why do you want to remove this block? I think we will be missing something if you do so. I can’t explain more as I am from my phone